package.json

```
@@ -19,7 +19,7 @@
         "papaparse": "^5.4.1",
         "prop-types": "^15.8.1",
         "ra-data-fakerest": "^5.1.1",
-        "ra-supabase": "^3.0.0",
+        "ra-supabase": "^3.0.1",
```

nitpick: there is no 3.0.1. You should require a 3.1.0 instead.


```suggestion
        "ra-supabase": "^3.1.0",
```